packages/rolldown/tests/fixtures/tree-shake/filter-expression/_config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineTest } from "rolldown-tests";
+import { and, code, exclude, id, not } from "rolldown/filterExpr";
+import { expect, vi } from "vitest";
+
+const transformHookFunction = vi.fn(() => {});
+export default defineTest({
+	config: {
+		plugins: [
+			{
+				name: "filter-expr",
+				transform: {
+					handler() {
+						transformHookFunction();
+					},
+					filter: {
+						custom: [exclude(and(id(/src/), not(code(/import\s+{/))))],
```

Comment:
This looks like so powerful and unlocks much potential abilities. Let's raise it up in the meeting and see how it goes.